openvdb/openvdb/io/Queue.cc

```diff
@@ -173,7 +176,10 @@ struct Queue::Impl
             std::placeholders::_1, std::placeholders::_2);
         task.setNotifier(notify);
         this->setStatus(task.id(), Queue::PENDING);
-        tbb::task::enqueue(task);
+
+        // get the global task arena
+        tbb::task_arena arena(tbb::task_arena::attach{});
+        arena.enqueue([task = std::move(task)] { task.execute(); });
```

Comment:
**Significant change**

Instead of queueing tasks through task::enqueue, add them to the global task arena.

@jmlait I think this is correct? i.e. `tbb::task_arena arena(tbb::task_arena::attach{});` gets a hooks to the "default" arena so I don't need to keep the arena in scope? This could be improved by having an arena per Queue, but I'm trying to keep the changes as minimal as possible.

Reply:
The change is faithful to what the code previously did. Offhand, I don't see an advantage to using an arena per queue?

Reply:
Thanks for confirming @e4lam. I'm trying to figure out what the better API would be, us wrapping the logging or delegating it to the caller. i.e:

```
Queue q;
q.concurrency(/*max threads*/1); // where each queue has its own configurable arena
q.write(...);
```
vs
```
// set arena for current scope
{
    tbb::task_arena arena(/*1*/);
    Queue q;
    q.write(...)
}
```

It's the same debate as the `bool threaded = true` argument that appears everywhere. In any case, not relevant for this PR.